<launch>
    <node pkg="my_assignment2" type="action_client_node" name="action_client" output="screen" />
    <node pkg="my_assignment2" type="target_service_node" name="target_service" output="screen" />
</launch>
